actions/feedback-service.js

```diff
@@ -101,6 +101,8 @@ function update(feedbackClass, cloudant, dbName, minrpi, maxrpi, activities) {
           }
           results.min_rpi = Infinity;
           results.max_rpi = -Infinity;
+          results = results.slice(0, 5);
```

Comment:
Wait... why is this not results.docs = results.docs.slice(0,5)? I don't remember the actual structure of this object.

Reply:
Yeah, good catch